decidim-core/app/views/layouts/decidim/header/_menu_breadcrumb_main_dropdown.html.erb

```diff
@@ -1,9 +1,10 @@
 <div id="<%= id %>" class="menu-bar__main-dropdown">
   <div class="menu-bar__main-dropdown__top">
-    <div class="menu-bar__main-dropdown__top-left">
-      <%= render partial: "layouts/decidim/header/menu_breadcrumb_main_dropdown_top_left" %>
+    <div class="mb-8">
+      <%= render partial: "layouts/decidim/header/menu_form_search_mobile" %>
     </div>
     <%= breadcrumb_root_menu.render %>
+    <%= render partial: "layouts/decidim/header/mobile_language_choose" %>
```

Comment:
This inclusion repeats the id `form-search-mobile` and `input-search-mobile` because the same view is being included in the view `_main.html.erb`.

If you want to reuse it I suggest to pass along the variables for the id:
```suggestion
    <%= render partial: "layouts/decidim/header/mobile_language_choose", locals: {id: "search-mobile"} %>
```
Add the variable too in _main.html.erb

Reply:
Done